nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-runtime/src/test/groovy/org/apache/nifi/NiFiGroovyTest.groovy

```diff
@@ -212,7 +211,7 @@ class NiFiGroovyTest extends GroovyTestCase {
     }
 
     private static NiFiProperties decrypt(NiFiProperties encryptedProperties, String keyHex) {
```

Comment:
Rename `keyHex` to `keyOrKeyId` as this could be AWS KMS key ID or AES key in hex.

Reply:
Renamed.